src/routes/users.js

```diff
@@ -50,15 +46,42 @@ router.post(
       user = await User.create({
         name,
         email,
-        password
+        password,
       });
 
       res.status(201).json(user.serialize());
     } catch (error) {
       console.error(error.message);
-      res.status(500).send("Server error");
+      res.status(500).send('Server error');
     }
-  }
+  },
 );
 
+/**
+ * @route GET api/users
+ * @access Public
+ * @desc Get all users
+ */
+router.get('/', async (req, res) => {
```

Comment:
I think we need to put jwtAuth here to protected to route. In such case we need to add the code below as well.
const jwtAuth = passport.authenticate('jwt', { session: false });
const passport=require('passport');